src/schema.rs

```diff
@@ -35,6 +35,11 @@ impl Schema {
 
     /// Returns an iterator over the imports of this [Schema].
     fn imports(&self) -> SchemaTypeIterator {
+        todo!()
+    }
+
+    /// Returns an iterator over the imported types of this [Schema].
+    fn import_types(&self) -> SchemaTypeIterator {
```

Comment:
From the perspective of an end user (who won't care much about ISL's `IMPORT_TYPE` grammar element), I think `imported_types` would fit expectations better:
```suggestion
    fn imported_types(&self) -> SchemaTypeIterator {
```